Lib/ufo2ft/outlineCompiler.py

```diff
@@ -583,7 +583,7 @@ def setupTable_OS2(self):
 
         def adjustOffset(offset, angle):
             """Adjust Y offset based on italic angle, to get X offset."""
-            return offset * math.tan(math.radians(-angle)) if angle else 0
+            return offset * math.atan(math.radians(-angle)) if angle else 0
```

Comment:
nit: 3 times is enough I would quesiton why we don't have a common function for this